ajax/itemTicket.php

```diff
@@ -69,10 +70,12 @@
                     foreach ($iterator as $data) {
                         if ($item_ticket->can($data['id'], DELETE)) {
                             $item_ticket->getFromDB($data['id']);
-                            $item_ticket->delete(['id' => $data['id']]);
+                            $deleted = $deleted && $item_ticket->delete(['id' => $data['id']]);
                         }
                     }
                 }
+            }
+            if ($deleted) {
                 unset($_POST['params']['items_id'][$_POST['itemtype']][array_search($_POST['items_id'], $_POST['params']['items_id'][$_POST['itemtype']])]);
```

Comment:
Is this containing an array of elements? Should this array be filtered more precisely, depending on each delete operation?